content/en/software/install-and-run/linux/_index.md

```diff
@@ -0,0 +1,116 @@
+﻿---
+title: Install and Run on Linux
+weight: 20
+type: docs
+---
+<style>.td-content blockquote { border-left: none; color: inherit; padding-left: 2rem;}</style>
+
+*Note that these instructions do not apply to Linux distros running within Windows System For Linux (WSL).  [Separate instructions are available for WSL here.](../windows/running-on-win-wsl).*
```

Comment:
Delete the period after the link `(../windows/running-on-win-wsl).`  That period and the period at the end of the text block, `[Separate instructions are available for WSL here.]` creates a sentence ending with 2 periods.